src/Tribe/Tickets_View.php

```diff
@@ -41,6 +41,7 @@ public static function hook() {
 		add_action( 'parse_request', [ $myself, 'prevent_page_redirect' ] );
 		add_filter( 'the_content', [ $myself, 'intercept_content' ] );
 		add_action( 'parse_request', [ $myself, 'maybe_regenerate_rewrite_rules' ] );
+		add_filter( 'tribe_events_views_v2_bootstrap_should_display_single', [ $myself, 'intercept_views_v2_single_display' ], 15, 4 );
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 75769450186e0a1b868f21640265f1df</sub>